### ISSUE TITLE
how to define a array:json type variable?

### ISSUE THREAD

**Issue by dong-king** [eez-open/studio#430]
**Describe the bug**
there is a `json` type in eez-studio v0.14.2.
As shown in the figure below, I defined a `array:json` type variable. But when checking, an error message is displayed.

![image](https://github.com/eez-open/studio/assets/58795990/5a0b365a-445d-416b-9f74-2a81f98acec7)

![image](https://github.com/eez-open/studio/assets/58795990/eea29464-9222-41fc-b847-a7b7ce0a2f3c)

**Comment by mvladic** [comment]
Indeed, this is a bug. But, when fixed you would need to set "Default value" as following:

```
[ json`{ "a": true, "b": 1 }`, json`{ "a": false, "b": 2 }` ]
```

In `Default value` field only literal values are allowed, so in this case you must use JSON literal, i.e. `` json`...` `` for each element.

But, you don't need to declare this variable as `array:json`, you can declare it as `json` as JSON value can also be an array. And you should set default value to:

```
json`[ { "a": true, "b": 1 }, { "a": false, b: 2 } ]`
```
